.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Do you know if this will work on windows?

Reply:
I reached out to the repository owner of Husky and this is what they mentioned:
`@CasuallyCaffeinated the only known issue is https://typicode.github.io/husky/#/?id=yarn-on-windows (it may have been fixed since). Otherwise no particular issue.`

`To avoid confusions, I'm going to lock this issue as it's an old one and concerns husky 4 which works quite differently than husky 7.`

So, I believe we should be good to go, even for those folks using a Windows environment. If they're using Yarn 2, and they're having issues, we can discuss more.

Reply:
Ok cool thanks. I have a windows machine (though I usually do development work on Linux or mac) and I'll test it out soon. 

Good job on this PR so far

Reply:
I also have a windows machine, but I do all my development work in a WSL 2 VM. Still, good catch since I didn't even think to double check that.